src/Compilers/CSharp/Portable/Emitter/Model/FunctionPointerSymbolsAdapter.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+#nullable enable
+
+using System.Reflection.Metadata;
+using Microsoft.Cci;
+using Microsoft.CodeAnalysis.Emit;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed partial class FunctionPointerTypeSymbol : Cci.IFunctionPointerTypeReference
```

Comment:
>FunctionPointerTypeSymbol [](start = 34, length = 25)

It looks like there is a typo in the name of the file - "FunctionPointerTypeSymbol**s**Adapter"